content/en/docs/instrumentation/net/automatic/_index.md

```diff
@@ -0,0 +1,275 @@
+
+# Run your application with instrumentation
+OTEL_SERVICE_NAME=myapp OTEL_RESOURCE_ATTRIBUTES=deployment.environment=staging,service.version=1.0.0 ./MyNetApp
+```
+
+{{% alert title="Note" color="note" %}} On macOS
+[`coreutils`](https://formulae.brew.sh/formula/coreutils) is required.
+{{% /alert %}}
+
+### Windows (PowerShell)
+
+On Windows, use the PowerShell module as an Administrator:
+
+```powershell
+# PowerShell 5.1 or higher is required
+# Download the module
+$module_url = "https://raw.githubusercontent.com/open-telemetry/opentelemetry-dotnet-instrumentation/v1.0.0-rc.2/OpenTelemetry.DotNet.Auto.psm1"
+$download_path = Join-Path $env:temp "OpenTelemetry.DotNet.Auto.psm1"
+Invoke-WebRequest -Uri $module_url -OutFile $download_path -UseBasicParsing
+
+# Import the module to use its functions
+Import-Module $download_path
+
+# Install core files (online vs offline method)
+Install-OpenTelemetryCore
+Install-OpenTelemetryCore -LocalPath "C:\Path\To\OpenTelemetry.zip"
+
+# Set up the instrumentation for the current PowerShell session
+Register-OpenTelemetryForCurrentSession -OTelServiceName "MyServiceDisplayName"
+
+# Run your application with instrumentation
+.\MyNetApp.exe
+
+# You can get usage information by calling the following commands
+
+# List all available commands
+Get-Command -Module OpenTelemetry.DotNet.Auto
+
+# Get command's usage information
+Get-Help Install-OpenTelemetryCore -Detailed
+```
+
+## Instrument a Windows Service running a .NET application
+
+Use the `OpenTelemetry.DotNet.Auto.psm1` PowerShell module to set up automatic
+instrumentation for a Windows Service:
+
+```powershell
+# Import the module
+Import-Module "OpenTelemetry.DotNet.Auto.psm1"
+
+# Install core files
+Install-OpenTelemetryCore
+
+# Set up your Windows Service instrumentation
+Register-OpenTelemetryForWindowsService -WindowsServiceName "WindowsServiceName" -OTelServiceName "MyServiceDisplayName"
+```
+
+{{% alert title="Note" color="note" %}}
+`Register-OpenTelemetryForWindowsService` performs a service restart.
+{{% /alert %}}
+
+### Configuration for Windows Service
+
+{{% alert title="Note" color="note" %}} Remember to restart the Windows Service
+after making configuration changes. You can do it by running
+`Restart-Service -Name $WindowsServiceName -Force` in PowerShell. {{% /alert %}}
+
+For .NET Framework applications you can configure the most common `OTEL_`
+settings (like `OTEL_RESOURCE_ATTRIBUTES`) via `appSettings` in `App.config`.
+
+The alternative is to set environment variables for the Windows Service in the
+Windows Registry.
+
+The registry key of a given Windows Service (named `$svcName`) is located under:
+
+```powershell
+HKLM\SYSTEM\CurrentControlSet\Services\$svcName
+```
+
+The environment variables are defined in a `REG_MULTI_SZ` (multiline registry
+value) called `Environment` in the following format:
+
+```env
+Var1=Value1
+Var2=Value2
+```
+
+## Instrument an ASP.NET application deployed on IIS
```

Comment:
Recommend a docs note under this section that makes it explicit that we're dealing with .NET Framework here